### ISSUE TITLE
Cheerio has been release to stable 1.0.0

### ISSUE THREAD

**Issue by ju-gow** [sveltech/svelte-filerouter#558]
### Describe the bug

1. updating dependencies
2. cheerio updates to 1.0.0
2. running dev or build not working anymore
see logs below

Thanks for all, routify is great !

### Reproduction

forcing "cheerio": "1.0.0-rc.12"
in devDependencies resolves bug

### Logs

```shell
$ pnpm run build

> svelte@0.0.0 build /path/to/project
> vite build

failed to load config from /path/to/project/vite.config.js
error during build:
file:///path/to/project/node_modules/.pnpm/@roxi+routify@3.0.0-next.232_@popperjs+core@2.11.8_spank@2.0.0_vite@5.4.0_terser@5.31.5_/node_modules/@roxi/routify/lib/buildtime/plugins/metaFromFile/metaFromFile.js:1
import cheerio from 'cheerio'
       ^^^^^^^
SyntaxError: The requested module 'cheerio' does not provide an export named 'default'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:123:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:191:5)
    at async ModuleLoader.import (node:internal/modules/esm/loader:336:24)
    at async loadConfigFromBundledFile (file:///path/to/project/node_modules/.pnpm/vite@5.4.0_terser@5.31.5/node_modules/vite/dist/node/chunks/dep-NjL7WTE1.js:66565:15)
    at async loadConfigFromFile (file:///path/to/project/node_modules/.pnpm/vite@5.4.0_terser@5.31.5/node_modules/vite/dist/node/chunks/dep-NjL7WTE1.js:66406:24)
    at async resolveConfig (file:///path/to/project/node_modules/.pnpm/vite@5.4.0_terser@5.31.5/node_modules/vite/dist/node/chunks/dep-NjL7WTE1.js:66014:24)
    at async build (file:///path/to/project/node_modules/.pnpm/vite@5.4.0_terser@5.31.5/node_modules/vite/dist/node/chunks/dep-NjL7WTE1.js:65111:18)
    at async CAC.<anonymous> (file:///path/to/project/node_modules/.pnpm/vite@5.4.0_terser@5.31.5/node_modules/vite/dist/node/cli.js:828:5)
 ELIFECYCLE  Command failed with exit code 1.
```

### System Info

```shell
System:
    OS: Linux 6.8 Ubuntu 24.04 LTS 24.04 LTS (Noble Numbat)
    CPU: (4) x64 Intel(R) Core(TM) i3-4010U CPU @ 1.70GHz
    Memory: 2.81 GB / 7.69 GB
    Container: Yes
    Shell: 5.2.21 - /bin/bash
  Binaries:
    Node: 18.19.1 - /usr/bin/node
    npm: 9.2.0 - /usr/bin/npm
    pnpm: 9.7.0 - /usr/local/bin/pnpm
  npmPackages:
    @roxi/routify: next => 3.0.0-next.232 
    svelte: ^4.2.15 => 4.2.18 
    vite: ^5.2.10 => 5.4.0
```

**Comment by jakobrosenberg** [comment]
Thank you for the heads up. This should be fixed now.

**Comment by alephpt** [comment]
I'm still experiencing this problem, even when switching to .233

Edit: Running `npm init routify@latest` and then seperately running `npm i @roxi/routify@3.0.0-next.233` did the fix.

**Comment by jakobrosenberg** [comment]
@alephpt can you check if it works with .234?

**Comment by alephpt** [comment]
@jakobrosenberg I'm getting weird errors for ReadableStream is not defined when starting dev server, when using .234.

Update, apparently reinstalling npm worked, but not for .234. For context, I'm running this via amazon fedora linux which has limitations on glibc versioning, which requires npm v16.

The long and short of it, .234 left me with the error, trying to install it 2 different ways:
Error [ERR_MODULE_NOT_FOUND]: Cannot find package '@roxi/routify' imported from /home/ec2-user/site/vite.config.js.timestamp-1726207247669-6e4031a6fd528.mjs

.. but this also looks like it's because the plugins are wanting a node version >= 18, and the amazon fedora limits to 16 without doing a bunch of system configurations. Rolling back to .142 seems to have worked with npm v16, and 233 seems to work with v18+

**Comment by alephpt** [comment]
@jakobrosenberg as an update, I migrated to amazon linux 2023, installed npm v22, and tried adding .233 like I did previously, and it am still getting `Error [ERR_MODULE_NOT_FOUND]: Cannot find package '@roxi/routify' imported from /home/ec2-user/site/vite.config.js` 

Not sure what changed but it's not working like it was before. Seems the latest release is broken good?

EDIT: As I said that, I deleted node_modules and reinstalled after changing to 233 and it works. Changing to .234 works as well.

Apologies for all the back and forth. It's been 2 days of fighting this. Happy to confirm it works!
